### DISCUSSION TITLE
Upload error

### DISCUSSION THREAD

**Discussion by Krusherk** [community/community#110974] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello I've been trying to upload a file but the page isn't working. Please fix and if you'd like to compensate me, that would be nice

**Comment by queenofcorgis** [comment]
Hi @Krusherk , we're happy you're here! You are more likely to get a useful response if you are explicit about what your error entails, giving a few more details might help someone give you a nudge in the right direction.
